cypress/test/tools/FiltersDropdown.cy.js

```diff
@@ -0,0 +1,32 @@
+import { mount } from 'cypress/react';
+import FiltersDropdown from '../../../components/tools/FiltersDropdown';
+
+describe('FiltersDropdown', () => {
+  const dataList = [
+    { name: 'Option 1' },
+    { name: 'Option 2' },
+    { name: 'Option 3' },
+  ];
+
+  it('should toggle options when clicked', () => {
+    const checkedOptions = [];
+    const setStateFunction = cy.stub().as('setStateFunction');
+
+    mount(
+      <FiltersDropdown
+        dataList={ dataList }
+        checkedOptions={ checkedOptions }
+        setStateFunction={ setStateFunction }
+      />
+    );
+
+    cy.get('[data-testid="FiltersDropdown-div"] > :nth-child(1)').click();
```

Comment:
Use `trigger` to click the options instead of `click`.

Reply:
done